src/main/frontend/commands.cljs

```diff
@@ -297,7 +297,12 @@
      ["Embed Youtube timestamp" [[:youtube/insert-timestamp]]]
 
      ["Embed Twitter tweet" [[:editor/input "{{tweet }}" {:last-pattern command-trigger
-                                                          :backward-pos 2}]]]]
+                                                          :backward-pos 2}]]]
+
+     ["Code block" [[:editor/input "```\n```\n" {:type            "block"
+                                                 :backward-pos    5
+                                                 :only-breakline? true}]
```

Comment:
Does this option make any functional difference or is just for munging newlines for certain environments? I disabled it locally, reloaded and command still seemed to work fine